ios/test_app.rb

```diff
@@ -53,7 +53,8 @@ def nearest_node_modules(project_root)
 
 def package_version(package_path)
   package_json = JSON.parse(File.read(File.join(package_path, 'package.json')))
-  package_json['version'].match(/(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)/)
+  match = package_json['version'].match(/(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)/)
+  Gem::Version.new(match[0])
```

Comment:
Why is there still a need to match at all, instead of `Gem::Version.new(package_json['version'])`?

Reply:
We don't. I just missed it :smile:

Reply:
Hah, perfect 👍